lib/jekyll-contentful/content_types.rb

```diff
@@ -78,7 +78,7 @@ def parse_reference_field
 
           def space
             @space ||= begin
-              management.spaces.find(ENV['CONTENTFUL_SPACE_ID'])
+              management.environments(ENV['CONTENTFUL_SPACE_ID']).find(ENV['CONTENTFUL_ENV'] || 'master')
```

Comment:
So will this always find `master`? Do we need to get `int` or `demo`?

Reply:
if it finds a CONTENTFUL_ENV it will use that OR it will default to master